_tools/generate.js

```diff
@@ -1,53 +1,148 @@
 const fs = require("fs").promises;
 const path = require("path");
 
-const { save, getTemplate, insert } = require("./utils");
+const { save, getTemplate, insert, DefaultListMap } = require("./utils");
 
 start();
 
 async function start() {
+  await Promise.all([generate("norge"), generate("sverige")]);
+}
+
+async function generate(country) {
   try {
     const template = await getTemplate("index_template");
-    const documents = await getDocuments();
-    const documentItems = documents.map(createItem).join("\n");
+    const documents = await getDocuments(country);
+
+    const headerList = [
+      createCountryRadioGroup(country),
+      createCollapsibleLists(documents),
+    ];
 
     const result = insert(template, {
-      content: documentItems
+      content: headerList.join("\n"),
     });
 
-    save("index", result);
+    save(country === "norge" ? "index" : "sverige", result);
 
-    console.log("Generated index.html");
+    console.log(`Generated ${country}.html`);
   } catch (e) {
     console.error("Error:", e);
   }
 }
 
+function createCountryRadioGroup(country) {
+  const norgeChecked = country === "norge" ? "checked" : "";
+  const redirectToSverige =
+    country === "norge" ? "onclick=\"event.preventDefault(); window.location.href='/sverige';\"" : "";
+  const sverigeChecked = country === "sverige" ? "checked" : "";
+  const redirectToNorge =
+    country === "sverige" ? "onclick=\"event.preventDefault();window.location.href = '/';\"" : "";
+
+  return `
+    <div class="countryRadioGroup">
+      <label class="toggleOption toggleOptionLeft">
+        <input type="radio" class="countryRadioButton" value="norge" ${norgeChecked} ${redirectToNorge}> Norge
+      </label>
+
+      <label class="toggleOption toggleOptionRight">
+        <input type="radio" class="countryRadioButton" value="sverige" ${sverigeChecked} ${redirectToSverige}> Sverige
+      </label>
+    </div>
```

Comment:
Her tror jeg egentlig det er like enkelt og mer korrekt å bruke `<a>`. Da får du mulighet til høyre museklikk -> åpne i ny tab, og det funker uansett hvor en er. Og det er kanskje mer semantisk korrekt. Trenger ikke henge seg så opp i at det ser ut som radio-buttons, for det er egentlig bare en visualisering.

UU-messig kan det vurderes om f.eks `aria-current="page"` er korrekt.